### PR TITLE
propose using ADRs

### PR DESCRIPTION
**Pull request by JaredCorduan**
cc @abailly-iohk

### CODE REVIEW

**Review by lehins — APPROVED** [review]
:+1:

File: docs/adr/2022-12-01_001-record-architectural-decisions.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+The [Hydra team](https://github.com/input-output-hk/hydra)
+has had success with the solution described below.
+See the [Hydra ADR 1](https://github.com/input-output-hk/hydra/blob/master/docs/adr/2021-06-05_001-record-architectural-decisions.md).
+
+## Decision
+
+* We will use _Architecture Decision Records_, as described by Michael Nygard in this [article](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions).
+* We will follow the convention of storing those ADRs as Markdown formatted documents stored under `docs/adr` directory, as exemplified in Nat Pryce's [adr-tools](https://github.com/npryce/adr-tools). This does not imply we will be using `adr-tools` itself.
+
+## Consequences
```
Comment (JaredCorduan) [inline docs/adr/2022-12-01_001-record-architectural-decisions.md:28]: I just realized that I misinterpreted this section. It's the immediate consequences, not to be confused with the "experience report" that can come after. I'll ~steal~ copy the prose from the Hydra ADR 1 :grin: .

**Review by aniketd — APPROVED** [review]
:+1:

**Review by teodanciu — APPROVED** [review]
Looks good to me, it looks like something that will pay off in different ways.. even writing the idea down makes you think better, not to mention share better and keep track of why decisions where made. Great !

**Pull request merged**